build-deno

```diff
@@ -5,20 +5,24 @@ set -exuo pipefail
 rm -rf deno; mkdir deno
 cp -rp src/* deno
 
+PACKAGE_VERSION=$(node -p 'require("./package.json").version')
+
 cat << EOF > deno/README.md
 # OpenAI Node API Library - Deno build
 
-This is a build produced from openai/openai-node - please go there to read the source, file issues, etc.
+This is a build produced from https://github.com/openai/openai-node - please go there to read the source and docs, file issues, etc.
 
-Installation:
+Usage:
 
-\`\`\`
-import OpenAI from "https://deno.land/x/openai";
+\`\`\`ts
+import OpenAI from "https://deno.land/x/openai@$PACKAGE_VERSION/mod.ts";
```

Comment:
Can you also update the README of openai-node? That has Deno examples, and we should update them to use `mod.ts` + point to this repo.

Reply:
ah right of course, ty